src/cpp/rstudio-tests.in

```diff
@@ -8,6 +8,14 @@ checkUnitTestFailure() {
    fi
 }
 
+checkRUnitTestFailure() {
+    FAILURES=$(cat ${CMAKE_CURRENT_BINARY_DIR}/testthat-failures.log)
```

Comment:
nit: I realize we don't do this consistently but we should really quote our shell expansions

Reply:
I can add quotes, but this variable will be resolved to an absolute path by our cmakefiles.

Reply:
What if my user account name has spaces within? :-)

(I made this mistake once before on Windows and unfortunately lots of things fall over in that case)